packages/cloudflare/src/cli/build/patches/investigated/copy-env-files.ts

```diff
@@ -0,0 +1,18 @@
+import fs from "node:fs";
+import path from "node:path";
+
+import { BuildOptions } from "@opennextjs/aws/build/helper.js";
+
+import { extractProjectEnvVars } from "../../utils";
+
+/**
+ * Copies the values extracted from the project's env files to the output directory for use in the worker.
+ */
+export function copyEnvFiles(options: BuildOptions) {
+  ["production", "development", "test"].forEach((mode) =>
```

Comment:
it looks like we need the `.local` version for `wrangler dev` too
ref https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#environment-variable-load-order

Reply:
it's extracting the local ones in the `extractProjectEnvVars` function